Sources/main.cpp

```diff
@@ -99,6 +101,12 @@ static int PlatformMain(std::string const &host, std::string const &port) {
 
   std::unique_ptr<Socket> serverSocket = CreateSocket(host, port, false);
 
+#if defined(OS_POSIX)
```

Comment:
I don't like using more OS ifdefs than we need. Could you just add a dummy function for non-posix, and return an error if someone tries to use daemonize?